src/components/EventInfo/EventInfo.jsx

```diff
@@ -1,36 +1,46 @@
 import React from "react"
+import styled from "styled-components"
+import * as styleVars from "../variable"
 
-import EventInfoStyles from "./EventInfoStyles"
 import Event49Img from "../../assets/images/event/McHacks-49.jpg"
 import Container from "./Container"
 import Image from "./Image"
 import Text from "./Text"
 
+const Seperator = styled.div`
```

Comment:
I would recommend that we move this into it's own file for clarity.

Reply:
True